contracts/token/ERC777/ERC777.sol

```diff
@@ -408,10 +409,13 @@ contract ERC777 is Context, IERC777, IERC20 {
         emit Transfer(from, to, amount);
     }
 
+    /**
+     * @dev See {ERC20-_approve}.
+     *
+     * Note that accounts cannot have allowance issued by their operators.
```

Comment:
We're missing the "Requirements" section here. Did you leave it out because it's already in ERC20? I would still put them here I think, especially because the variables are named differently.

Reply:
I left it out because it's on ERC20 already (the same applies to `transfer`, `transferFrom` and friends).

The doc improvements will be carried over once we merge the 2.5.1 release branch into master, along with the changelog notice, etc.